ports/minitest/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO Atliac/minitest
+    REF v1.1.1

```

```suggestion
    REF "v${VERSION}"
```
You can use the new function. :)